src/api/swagger/restapi/operations/releases/create_release.go

```diff
@@ -84,6 +84,10 @@ type CreateReleaseBody struct {
 	/* Name of the new release
 	 */
 	ReleaseName string `json:"releaseName,omitempty"`
+
+	/* Values of the new release
+	 */
+	Values map[string]string `json:"values,omitempty"`
```

Comment:
This will only deal with top-level values. What if we just make this a raw string and just assume it is correctly formed. The Helm client takes a string anyway, so we wouldn't need to convert the map to a string, and it should return an error if this is badly formed.

Reply:
I would do that . But how to prevent or escape values including = and , ???